src/pages/Projects/components/RoleModal/RoleModal.module.css

```diff
@@ -7,15 +7,15 @@
   opacity: 0.5;
 }
 
-.assignmentSelect div[class$="ValueContainer"] {
+.assignmentSelect div[class*="value-container"] {
```

Comment:
I didn't know about that, interesting. Trying it now but it doesn't seem to work here, I think it's because we're using react-select and it might be doing it differently. the inner element class names are generated randomly. for ex RoleModal_assignmentSelect__1vTkk__value-container RoleModal_assignmentSelect__1vTkk__value-container--has-value css-g1d714-ValueContainer.

 but they still always contain keywords like value-container, menu-list... so doing *= to select them should be consistent.

Reply:
Oh. I was assuming these were the full class names. I don't like this solution--it feels very brittle--but if we have no other choice, so be it. Could you add `className` to the components when you render them maybe?

Reply:
I made it work with `:global`, I wasn't doing it correctly before.